lib/src/dispatcher.ts

```diff
@@ -39,6 +45,11 @@ export class Dispatcher<sync extends 'sync' | 'async'> {
   // dispatching messages, this completes.
   private readonly messages$ = new Subject<proto.OutboundMessage>();
 
+  // Subject to unsubscribe from all outbound messages to prevent dispatchers
+  // that happen to reuse a compilation ID from receiving messages intended for
+  // past dispatchers.
```

Comment:
```suggestion
  // Subject to unsubscribe from all outbound messages to prevent past
  // dispatchers with compilation IDs reused by future dispatchers from
  // receiving messages intended for future dispatchers.
```